tanzu-build-service/build-image-action.md

```diff
@@ -11,89 +11,124 @@ Use this GitHub Action to creates a Tanzu Build Service Build on a cluster.
     ---
     apiVersion: v1
     kind: ServiceAccount
     metadata:
-      name: default
-      namespace: <developer-namespace>
-    secrets:
-      - name: image-secret
-    imagePullSecrets:
-      - name: image-secret
+       namespace: DEVELOPER_NAMESPACE
+       name: github-actions
     ---
-    apiVersion: v1
-    kind: Secret
+    apiVersion: rbac.authorization.k8s.io/v1
+    kind: ClusterRoleBinding
     metadata:
-      name: image-secret
-      namespace: <developer-namespace>
-      annotations:
-        secretgen.carvel.dev/image-pull-secret: ""
-    type: kubernetes.io/dockerconfigjson
-    data:
-      .dockerconfigjson: e30K
-    ```
-
-3. Create a service account in the developer namespace and bind it to the following roles:
-
-    ```yaml
+       name: github-actions
+    subjects:
+       - kind: ServiceAccount
+         namespace: DEVELOPER_NAMESPACE
+         name: github-actions
+    roleRef:
+       kind: ClusterRole
+       name: github-actions
+       apiGroup: rbac.authorization.k8s.io
+    ---
+    apiVersion: rbac.authorization.k8s.io/v1
+    kind: RoleBinding
+    metadata:
+       name: github-actions
+       namespace: DEVELOPER_NAMESPACE
+    subjects:
+       - kind: ServiceAccount
+         namespace: DEVELOPER_NAMESPACE
+         name: github-actions
+    roleRef:
+       kind: Role
+       name: github-actions
+       apiGroup: rbac.authorization.k8s.io
     ---
     apiVersion: rbac.authorization.k8s.io/v1
     kind: ClusterRole
     metadata:
-      name: github-actions
+       name: github-actions
     rules:
-      - apiGroups: [ 'kpack.io' ]
-        resources:
-          - clusterbuilders
-        verbs: [ 'get' ]
+       - apiGroups: ['kpack.io']
+         resources:
+            - clusterbuilders
+         verbs: ['get']
     ---
     apiVersion: rbac.authorization.k8s.io/v1
     kind: Role
     metadata:
-      name: github-actions
-      namespace: <developer-namespace>
+       name: github-actions
+       namespace: DEVELOPER_NAMESPACE
     rules:
-      - apiGroups: [ '' ]
-        resources: [ 'pods', 'pods/log' ]
-        verbs: [ 'get', 'watch', 'list' ]
-      - apiGroups: [ 'kpack.io' ]
-        resources:
-          - builds
-        verbs: [ 'get', 'watch', 'list', 'create', 'delete' ]
+       - apiGroups: ['']
+         resources: ['pods']
+         verbs: ['get', 'watch', 'list']
+       - apiGroups: ['']
+         resources: ['pods/log']
+         verbs: ['get']
+       - apiGroups: ['kpack.io']
+         resources:
+            - builds
+         verbs: ['get', 'watch', 'list', 'create', 'delete']
     ```
 
-4. This action needs access to the cluster, using the Kubernetes service account created, with the
-following [GitHub encrypted secrets](https://docs.github.com/en/actions/security-guides/encrypted-secrets) set:
-
-    - CA_CERT
-    - NAMESPACE
-    - TOKEN
-    - SERVER
+Then to access the values (on GKE, if using another IAAS, this may be different):
```

Comment:
This is the command to access the values in GKE - right?
So it should say:
>To access the values on GKE (if using another IAAS, this may be different):